Python/python.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+def rock_paper_scissors():
+    
+    turn_options = ['Rock', 'Paper', 'Scissors']
+
+    player_turn = input("Your turn: Choose Rock, Paper, or Scissors:") 
+
+    opponent_turn = random.choice(turn_options)
+
+    print("You chose:", player_turn)
+
+    
+
+    for x in player_turn:
+
+        if x != "Rock" or "Paper" or "Scissors":
+            print("Incorrect option for turn. Choose Rock, Paper, or Scissors")
+            print(input("Your turn: Choose Rock, Paper, or Scissors:"))
```

Comment:
No need to tell them they have the wrong move, just repeat the `Your Move: ` directive (like the other implementations):

https://github.com/zachflower/rock-paper-scissors/blob/master/php/rps.php#L15-L20